abcix/example/kvstore/kvstore.go

```diff
@@ -86,8 +93,86 @@ func (app *Application) Info(req types.RequestInfo) (resInfo types.ResponseInfo)
 }
 
 func (app *Application) CheckTx(req types.RequestCheckTx) types.ResponseCheckTx {
-	// Customize priority by user
-	return types.ResponseCheckTx{Code: code.CodeTypeOK, GasWanted: 1, Priority: 1}
+	var key string
+	// Priority can only be set by directly calling CheckTx, not by RecheckTx
```

Comment:
Since now priority is not deterministic, the working flow is like:
Total 5 txs, the first time calling CheckTx got `tx:priority` ->` tx1: 100, tx2: 86, tx3:90, tx4: 75, tx5: 9`
If we set each block can only include 3 txs, the first createBlock includes `[tx1, tx5, tx3] `, tx2 and tx4 still left in mempool
After executed block, commit will call RecheckTx, `tx2: 75, tx4: 89 `, u can see the priority is changed 
Then the next createBlock includes `[tx4, tx2]`, the tx order u can see is also changed, which is not the original tx order [`tx2, tx4]`